octoprint_prometheus_exporter/__init__.py

```diff
@@ -17,8 +17,6 @@ class PrometheusExporterPlugin(octoprint.plugin.BlueprintPlugin,
 							   octoprint.plugin.ProgressPlugin,
 							   octoprint.plugin.EventHandlerPlugin):
 
-	_registry = CollectorRegistry(auto_describe=True)
```

Comment:
Actually it is needed based on [this commit](https://github.com/tg44/OctoPrint-Prometheus-Exporter/commit/8776b472a9bbf6859caf8abc528bba9b41ecb8bd).